mteb/cli.py

```diff
@@ -354,6 +368,50 @@ def add_create_meta_parser(subparsers) -> None:
     parser.set_defaults(func=create_meta)
 
 
+def add_create_table_parser(subparsers) -> None:
+    parser = subparsers.add_parser(
+        "create-table", help="Create comparison tables for MTEB models"
+    )
+
+    parser.add_argument(
+        "--results",
+        type=str,
+        default="results/",
+        help="Path to fetch results from (local folder or GitHub repo URL)",
+    )
+
+    parser.add_argument(
+        "--models",
+        type=str,
+        default=None,
+        help="Comma-separated list of models to include in the table (default: all models)",
+    )
```

Comment:
I think it would be better to add `nargs` and use like `--models model1 model2` 
```suggestion
    parser.add_argument(
        "--models",
        type=str,
        nargs='*',
        default=None,
        help="Models to include in the table (default: all models from results dir)",
    )
```